core/models.py

```diff
@@ -432,6 +432,18 @@ def is_claim_admin(self):
         else:
             return False
 
+    @property
+    def is_imis_admin(self):
```

Comment:
I wonder if we could move such property (from this InteractiveUser model) to more generic model because the same method is defined in another model (User) (let me know if this is possible according to you - if no - drop this comment suggestion please) @lruzicki Maybe in the User could be sufficient to reuse in InteractiveUser too.